src/utils/daos_hdlr.c

```diff
@@ -608,6 +632,11 @@ cont_get_attr_hdlr(struct cmd_args_s *ap)
 	char				*buf;
```

Comment:
With the change to free buf at the out: label I think it needs to be initialized to NULL here